examples/waffle/e2e/test/dex.test.ts

```diff
@@ -99,52 +98,51 @@ describe('dex test', () => {
       value: '0x0'
     });
 
-    expect(hexlifyRpcResult(receipt)).deep.eq({
-      to: hexlifyRpcResult(TokenA.address),
+    expect(hexlifyRpcResult(receipt)).deep.eq(hexlifyRpcResult({
+      to: TokenA.address,
       from: aliceEvmAddress,
       contractAddress: null,
-      transactionIndex: hexlifyRpcResult(tx.transactionIndex),
-      gasUsed: hexlifyRpcResult(receipt.gasUsed),
-      logsBloom: hexlifyRpcResult(receipt.logsBloom),
+      transactionIndex: tx.transactionIndex,
+      gasUsed: receipt.gasUsed,
+      logsBloom: receipt.logsBloom,
       blockHash: tx.blockHash,
       transactionHash: txHash,
       logs: [
         {
-          transactionIndex: hexlifyRpcResult(tx.transactionIndex),
-          blockNumber: hexlifyRpcResult(tx.blockNumber),
+          transactionIndex: tx.transactionIndex,
+          blockNumber: tx.blockNumber,
           transactionHash: txHash,
-          address: hexlifyRpcResult(TokenA.address),
+          address: TokenA.address,
           topics: [
             '0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef',
             '0x00000000000000000000000082a258cb20e2adb4788153cd5eb5839615ece9a0',
             '0x000000000000000000000000905c015e38c24ed973fd6075541a124c621fa743'
           ],
           data: '0x000000000000000000000000000000000000000000000000000000000000000a',
-          logIndex: hexlifyRpcResult(0),
+          logIndex: 0,
           blockHash: tx.blockHash
         },
         {
-          transactionIndex: hexlifyRpcResult(tx.transactionIndex),
-          blockNumber: hexlifyRpcResult(tx.blockNumber),
+          transactionIndex: tx.transactionIndex,
+          blockNumber: tx.blockNumber,
           transactionHash: txHash,
-          address: hexlifyRpcResult(TokenB.address),
+          address: TokenB.address,
           topics: [
             '0xddf252ad1be2c89b69c2b068fc378daa952ba7f163c4a11628f55a4df523b3ef',
             '0x000000000000000000000000905c015e38c24ed973fd6075541a124c621fa743',
             '0x00000000000000000000000082a258cb20e2adb4788153cd5eb5839615ece9a0'
           ],
           data: '0x0000000000000000000000000000000000000000000000000000000000000062',
-          logIndex: hexlifyRpcResult(1),
+          logIndex: 1,
           blockHash: tx.blockHash
         }
       ],
-      blockNumber: hexlifyRpcResult(tx.blockNumber),
-      cumulativeGasUsed: hexlifyRpcResult(receipt.cumulativeGasUsed),
-      effectiveGasPrice: hexlifyRpcResult(receipt.effectiveGasPrice),
+      blockNumber: tx.blockNumber,
+      cumulativeGasUsed: receipt.cumulativeGasUsed,
+      effectiveGasPrice: receipt.effectiveGasPrice,
       status: '0x1',
       type: '0x0',
-      byzantium: true,
```

Comment:
https://github.com/AcalaNetwork/bodhi.js/blob/ecdb246879d97dcdb2b8c004712bc519643f219b/eth-rpc-adapter/src/eip1193-bridge.ts#L340

Removing the `byzantium` field causes exception here

Reply:
I think it won't cause exception, but let's remove this extra line
<img width="173" alt="Screenshot 2023-02-19 at 13 17 07" src="https://user-images.githubusercontent.com/16548786/219923611-5036428e-5a7b-42e4-bfa9-fa3b1eee5fbb.png">